hack/lib/util.sh

```diff
@@ -457,7 +457,7 @@ kube::util::godep_restored() {
 kube::util::ensure_clean_working_dir() {
   if ! git diff --exit-code; then
     echo -e "\nUnexpected dirty working directory."
-    exit 1
+    return 1
```

Comment:
Although we usually have `set -e` do we really want to rely on that? I'm fine either way, but it does seem like the original `exit 1` would be more reliable...